src/pages/topics/[slug].tsx

```diff
@@ -49,32 +57,82 @@ const Topic: NextPage<TopicProps> = ({
+                    paddingTop: ['inc40', null, 'inc50', 'inc70'],
+
+                    rowGap: ['inc90', null, null, 'inc130'],
+                }}
+            >
+                {(variant !== 'light' || topics.length > 0) && (
+                    <div>
+                        <GridLayout
+                            sx={{ rowGap: ['inc90', null, null, 'inc130'] }}
+                        >
+                            {topics.length > 0 && (
+                                <TopicCardsContainer
+                                    topics={topics}
+                                    title={`${name} Topics`}
+                                />
+                            )}
+                            {variant !== 'light' && (
+                                <FeaturedCardSection content={featured} />
+                            )}
+                            {variant === 'heavy' && (
+                                <>
+                                    {tutorials.length > 2 && (
+                                        <CardSection
+                                            content={tutorials}
+                                            title="Tutorials"
+                                        />
+                                    )}
+                                    {demoApps.length > 2 && (
+                                        <CardSection
+                                            content={demoApps}
+                                            title="Demo Apps"
+                                        />
+                                    )}
+                                    {articles.length > 2 && (
+                                        <CardSection
+                                            content={articles}
+                                            title="Articles"
+                                        />
+                                    )}
+                                    {videos.length > 2 && (
+                                        <CardSection
+                                            content={videos}
+                                            title="Videos"
+                                        />
+                                    )}
+                                    {podcasts.length > 2 && (
+                                        <CardSection
+                                            content={podcasts}
+                                            title="Podcasts"
+                                            direction="column"
+                                        />
+                                    )}
+                                </>
+                            )}
+                        </GridLayout>
+                    </div>
+                )}
+                <Search name={name} slug={slug} />
+
+                {variant === 'light' && relatedTopics.length > 0 && (
+                    <div>
```

Comment:
do we need this `div` here?

Reply:
I was having some issues with `GridLayout` having some weird margins, and we wrapped the upper topics container grid in a div as well, so I just did that here. But I'll see if I can sort it out because yeah having excessive elements is annoying.